internal/yml/yml_test/compare_test.go

```diff
@@ -132,3 +134,145 @@ foo:
 		})
 	}
 }
+
+func TestEqualWithExclusion(t *testing.T) {
```

Comment:
In the land of yaml, we can use flow style syntax, which is kind of a lightweight JSON, which allows us to keep indentation so that the files stay 🥨 (closest emoji to "pretty" was "pretzel")

```go
			yaml1: `
foo:
  bar: baz
  nested:
    - item1
    - item2
`,
```

Becomes (or even nicer because I am struggling with github formatting in the rich editor):

```go
			yaml1: `{
				foo: {
				   bar: baz,
				   nested: [item1, item2],
				},
			}`
```